[2025-01-29T17:13:53.231+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T17:13:53.257+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2024-12-08T00:00:00+00:00 [queued]>
[2025-01-29T17:13:53.284+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2024-12-08T00:00:00+00:00 [queued]>
[2025-01-29T17:13:53.291+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-29T17:13:53.337+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): taskA> on 2024-12-08 00:00:00+00:00
[2025-01-29T17:13:53.355+0000] {standard_task_runner.py:72} INFO - Started process 2855 to run task
[2025-01-29T17:13:53.363+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'cron_catchup_backfill', 'taskA', 'backfill__2024-12-08T00:00:00+00:00', '--job-id', '179', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpudlikolo']
[2025-01-29T17:13:53.367+0000] {standard_task_runner.py:105} INFO - Job 179: Subtask taskA
[2025-01-29T17:13:53.458+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskA backfill__2024-12-08T00:00:00+00:00 [running]> on host ef9d5dc750d2
[2025-01-29T17:13:53.604+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2024-12-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-12-08T00:00:00+00:00'
[2025-01-29T17:13:53.610+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T17:13:53.635+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-29T17:13:53.640+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK A has executed']
[2025-01-29T17:13:53.655+0000] {subprocess.py:99} INFO - Output:
[2025-01-29T17:13:53.658+0000] {subprocess.py:106} INFO - TASK A has executed
[2025-01-29T17:13:53.659+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-29T17:13:53.707+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T17:13:53.709+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskA, run_id=backfill__2024-12-08T00:00:00+00:00, execution_date=20241208T000000, start_date=20250129T171353, end_date=20250129T171353
[2025-01-29T17:13:53.741+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T17:13:53.777+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T17:13:53.782+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
